README.md

```diff
@@ -4,26 +4,30 @@
 
 Scripts to automate the release process.
 
-These scripts automate the release process which to-date has been
-a tedious manual process prone to mistakes. The script performs steps 1
-through 8.
-
-1. Check-out a current ``master`` branch
-2. Create a ``release-candidate`` branch
-3. Hard reset ``release-candidate`` to ``master``
-4. Generate release notes
-5. Update version numbers and ``RELEASE.rst``
-6. Commit updates and push ``release-candidate`` branch
-7. Generate release notes with checkboxes
-8. Open PR to merge ``release-candidate`` branch into ``release`` branch
-9. Merge PRs once developers verify their commits (manual step)
-10. Merge the ``release`` branch into the ``master`` branch,
-    and push ``master`` to ``origin`` (manual step)
-11. Send email notifications (manual step)
+A full release can be performed by following the steps below:
+
+1. Run release.sh to create a PR for the release (see details in the 
+    ["How to use release.sh"](#how-to-use-releasesh) section below)
+1. Inform the team that a release PR is up and that they need to verify their commits
+1. Once developers verify their commits, merge the 'release-candidate' branch into 'release', and push
+    the release branch.
+    *(NOTE: This can be done using the 'Merge Pull Request' button in the Github PR issue. Do not 
+    delete the branch after merging)*
+1. Tag the ``release`` branch with the version number and push the tag to
+    the remote. For example, these commands create an annotated tag for
+    version 0.3.0 and push the tag to the remote. (You don't need to specify
+    the sha of the commit if you're on the branch you want to tag.)
+    ```
+    git tag -a -m "Release 0.3.0" v0.3.0 <sha of commit>
+    git push --follow-tags
+    ```
```

Comment:
For some reason this block is rendering inline. I think maybe it needs a newline between lines 19 and 20.